layer demo/layer-demo.bas

```diff
@@ -30,6 +35,11 @@
 140 IF 8*I >255 THEN POKE $9F21,$81             : REM IF > 255, INC HI BYTE
 150 FOR J=1 TO 8 :POKE $9F23,$FF : NEXT J       : REM WRITE 8 BYTES PER TILE
 160 NEXT I
+200 REM
+201 REM == Fill the tile map with blank tiles ==
+202 REM
+210 REM POKE $9F20,0:POKE $9F21,$40:POKE $9F22,$10
+220 REM FOR I=1 TO 64*128 : POKE $9F23,81: POKE $9F23,0 : NEXT I
```

Comment:
Zeroing out was an issue, so I wrote some code.  After doing so, I realized why op-codes were directly written and jumped to in order to fill characters in the original code.  I decided to do the same thing, but left this in comments for people to follow.